### DISCUSSION TITLE
Ingest the whole notes

### DISCUSSION THREAD

**Discussion by m0d02x** [logancyang/obsidian-copilot#191] [General]
Hello!

Rather than just "active notes", is it possible for us to load the whole MD/Notes and start asking about it?

I tried but looks like the answers work well only for activate notes.

Also, on LocalAI, im using [LM Studio](lmstudio.ai) its easier to setup LocalAI that way.

**Comment by logancyang** [comment]
I'm going to have some major updates to enable multiple notes as context in different ways.

LM Studio looks good, will look into it!

**Comment by genericrich** [comment]
modo2x, did you have to do anything to get LM Studio to work? I am pointing copilot at it but it looks like the API Copilot is using is different somehow?

**Reply by m0d02x** [reply]
![CleanShot 2023-12-08 at 05 43 42@2x](https://github.com/logancyang/obsidian-copilot/assets/151829413/565f6d5e-3f14-477d-873e-487a27abfdab). We can see it is hitting the server and stream the response.

Yeah, managed to get it to work.

Setting on the CoPilot

![CleanShot 2023-12-08 at 05 44 57@2x](https://github.com/logancyang/obsidian-copilot/assets/151829413/1654df95-2287-4ce2-bd0d-6408618ac1c8)

![CleanShot 2023-12-08 at 05 45 12@2x](https://github.com/logancyang/obsidian-copilot/assets/151829413/f31cae34-ed09-4258-8960-f79d606abd81)
